### PR TITLE
Fix Rider Test Suites 

### PR DESCRIPTION
**Pull request by vchikoti1998**
## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
This tests are failing because not picking the correct solutionFile. So corrected the path of solution file.
## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt [hunk 1]
```diff
@@ -49,20 +51,17 @@ class JavaLocalLambdaRunConfigurationIntegrationTest(private val runtime: Lambda
...
+        mkdirs()
+    }
 
     @Before
     fun setUp() {
         setSamExecutableFromEnvironment()
-
-        val gradleUserHome = temporaryFolder.newFolder("gradle-user-home")
-        System.setProperty("gradle.user.home", gradleUserHome.absolutePath)
+        System.setProperty("gradle.user.home", customGradleUserHome.absolutePath)
```
Comment (rli) [inline plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt]: as discussed offline, this sets the property for the current running gradle process, which in effect does not do anything.

this needs to be set directly on the daemons of test projects being invoked, either through a `gradle.properties` file, or by somehow passing it as an env var through SAM CLI

Comment (vchikoti1998) [reply plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt]: working on separate [PR] (https://github.com/aws/aws-toolkit-jetbrains/pull/4549)

**Review by rli — APPROVED** [review]

**Pull request merged**